manifests/integrations/twemproxy.pp

```diff
@@ -0,0 +1,42 @@
+# Class: datadog_agent::integrations::twemproxy
+#
+# This class will install the necessary configuration for the twemproxy aka nutcracker integration
+#
+# Parameters:
+#   $host:
+#       The host twemproxy is running on. Defaults to '127.0.0.1'
+#   $port
+#       The twemproxy password for the datadog user. Defaults to 22222
+#
+# Sample Usage:
+#
+#  class { 'datadog_agent::integrations::twemproxy' :
+#    servers => [
+#      {
+#        'host' => 'localhost',
+#        'port' => '22222',
+#      },
+#      {
+#        'host' => 'localhost',
+#        'port' => '22223',
+#      },
+#    ]
+#  }
+#
+class datadog_agent::integrations::twemproxy(
+  $servers = [{'host' => 'localhost', 'port' => '22222'}]
```

Comment:
I'd rather respect the pattern we have in similar checks where we have arguments `host` and `port` and maybe some basic options (for single instance) and then an `instances` argument for users who wish to specify multiple instances for the check. Don't get me wrong, this is perfectly fine, but it breaks a pattern used widely across the check. 

I am aware that `mongo` and `zk` both follow this approach you have here, but I'd rather have that be the exception. Most integrations follow the pattern you can see `haproxy`. If you could fix that up, and the `erb`, which should be easy, that would be great.